buildSrc/src/main/kotlin/buildsrc/conventions/kotlin-multiplatform.gradle.kts

```diff
@@ -0,0 +1,68 @@
+package buildsrc.conventions
+
+/** Kotlin/Multiplatform convention that configures all targets for Parsus */
+
+plugins {
+    id("buildsrc.conventions.kotlin-multiplatform-base")
+}
+
+kotlin {
+    jvm()
+
+    js(IR) {
+        browser()
+        nodejs()
+    }
+
+    linuxX64()
+
+//    mingwX64() // TODO find a workaround/replacement for AssertK missing a mingw variant
```

Comment:
[AssertK doesn't support MingGW](https://github.com/willowtreeapps/assertk/issues/406). I'd be happy to migrate to use Kotest Assertions (but still use kotlin.test annotations), if you approve?

Reply:
Hopefully the next release of AssertK will support Windows 🎉 

https://github.com/willowtreeapps/assertk/pull/439